cmd/utils.go

```diff
@@ -78,3 +81,41 @@ func filterYaml(cmd *cobra.Command, args []string, toComplete string) ([]string,
 	}
 	return []string{"yaml", "yml"}, cobra.ShellCompDirectiveFilterFileExt
 }
+
+var flagSkipGroups []string
+var flagOnlyGroups []string
+
+func addGroupSelectionFlags(c *cobra.Command) *cobra.Command {
+	c.Flags().StringSliceVar(&flagSkipGroups, "skip", nil, "Skip groups with the given names")
+	c.Flags().StringSliceVar(&flagOnlyGroups, "only", nil, "Only apply to groups with the given names")
+	return c
+}
+
+func validateGroupSelectionFlags(bp config.Blueprint) error {
+	if flagOnlyGroups != nil && flagSkipGroups != nil {
+		return errors.New("cannot specify both --only and --skip")
+	}
+
+	dict := []string{}
+	for _, group := range bp.Groups {
+		dict = append(dict, string(group.Name))
+	}
+
+	for _, g := range append(flagOnlyGroups, flagSkipGroups...) {
+		if !slices.Contains(dict, g) {
+			return config.HintSpelling(g, dict, fmt.Errorf("group %q not found", g))
+		}
+	}
```

Comment:
Probably overkill but wouldn't storing the only/skip groups in a map be faster for lookup?  Can HintSpelling use keys from a map?

Reply:
On such small numbers of elements linear search hash ~same latency (if not better) than hashing.
Keeping it as is for the sake of simplicity.